program-workflows/adjust-recency-data-sync-frequency.md

```diff
@@ -0,0 +1,21 @@
+##HOW FREQUENTLY TO SEND RECENCY DATA
+
+This guide will show how to adjust the frequency of sending data. This can be useful in low connectivity facilities where internet is only available for short period. Reducing the frequency of sending data will ensure internet availability is utilized.
+
+1. Click legacy administration link
+
+![Legacy Admin Button](../images/legacy-admin-button.png)
+
+2. In the Maintenance section click Manage Scheduler
+
+![Manage Scheduler](../images/manage-scheduler.png)
+
+3.	Click the Send recency data to central server link on the list of tasks. Here you will also be able to see when the next attempt to send data will be.
+
+![Send Recency Data task](../images/send-recency-data-to-a-central-server.png)
+
+4.	Reduce the repeat interval to ten minutes.
```

Comment:
@ODORA0 Why are you changing it to 10 minutes? Better to recommend changing this 2 hours rather than shorter

Reply:
@ssmusoke 10 mins for facilities with a poor internet connection so that any slight connection will allow data to sync through, atleast that's what we've been doing for these type of facilities